### ISSUE TITLE
Private folder reset request

### ISSUE THREAD

**Issue by DopefishJustin** [keybase/client#25253]
Looks like I have a similar issue to #22507, #13751, #21609, etc. I am unable to load file listings for `/keybase/private/dopefishjustin` on the Windows or Android clients, sometimes with a nonexistent block error. Other folders are OK. Started years ago but I never got around to reporting it. Not sure if there are still any staff watching this tracker but if so, it would be swell if I could get a server-side reset please. I have backups of all data.

Signed request:
```
BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkCXdvVH thVIdoK0t
KcipCl J2t3xwPTEeskFe7 wHhbvmM7J0VFJiH WPbDvh2ZsBkk3B9 Unt3WZqJfH4mjiS GEKgTl0KA
RYjZna DWmO0iynKjzO7hN eRQpLjtg625I8sT 3Dgm85ZIoijHA9V TnjQZxkTHVnR658 FQLO6vNs2
FV3xJk cv5mz7FlHgHwbSE DtQ0XRMY7R6Xr7X ONhZ7bZdQ8GhL5Z HSbg80GkE90ogX5 spBBElYy3
2lzL4k vMxpOBIkXdYpYqC 1uZub3595uoAuBa SNo2Lvhn5JaNDLB wX7MWE0Po1DGxzH i6aSrK3SA
ItsrF9 t1GFjoZG7lhFRm. END KEYBASE SALTPACK SIGNED MESSAGE.
```

Log ID is 244b79631f44003ed4fa621c

Thanks!

**Comment by DopefishJustin** [comment]
Rescinding this request as I have managed to get it working again. After installing on Linux I noticed I could access the filesystem, although in an older state. Shutting down the Windows client and deleting the kbfs* folders in %LOCALAPPDATA%\Keybase put the Windows client into the same state and I am now able to synchronize changes between clients as expected. There seems to have been an enormous amount of data in kbfs_journal that was not going up to the server. (Luckily I did not need the data.)

**Comment by DopefishJustin** [comment]
Rescinding this request as I have managed to get it working again. After installing on Linux I noticed I could access the filesystem, although in an older state. Shutting down the Windows client and deleting the kbfs* folders in %LOCALAPPDATA%\Keybase put the Windows client into the same state and I am now able to synchronize changes between clients as expected. There seems to have been an enormous amount of data in kbfs_journal that was not going up to the server. (Luckily I did not need the data.)
